manifests/c/cobalt-org/cobalt/0.19.6/cobalt-org.cobalt.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: cobalt-org.cobalt
+PackageVersion: 0.19.6
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: cobalt.exe
+  PortableCommandAlias: cobalt
+InstallModes:
+- silent
+- silentWithProgress
+UpgradeBehavior: uninstallPrevious
+Commands:
+- cobalt
+ReleaseDate: 2024-08-07
+Installers:

```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Reply:
Hello @stephengillie ! I'm new to WinGet packaging and am unsure how to replicate this? On my system the package installs and works just fine with `winget install --manifest <snip>/manifests/c/cobalt-org/cobalt/0.19.6/`. What am I doing wrong?